nestkernel/syn_id_delay.h

```diff
@@ -72,15 +72,15 @@ struct SynIdDelay
   }
 
   void
-  set_has_source_subsequent_targets( const bool subsequent_targets )
+  set_source_has_more_targets( const bool more_targets )
   {
-    this->subsequent_targets = subsequent_targets;
+    this->more_targets = more_targets;
```

Comment:
Is the `this->` needed here?

Reply:
i think so, since the parameter name is the same as the member name and we need to distinguish the two

Reply:
OK!